aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -151,20 +153,66 @@ module aptos_framework::block {
         stake::update_performance_statistics(proposer_index, failed_proposer_indices);
         state_storage::on_new_block(reconfiguration::current_epoch());
 
-        if (timestamp - reconfiguration::last_reconfiguration_time() >= block_metadata_ref.epoch_interval) {
+        block_metadata_ref.epoch_interval
+    }
+
+    /// Set the metadata for the current block.
+    /// The runtime always runs this before executing the transactions in a block.
+    fun block_prologue(
+        vm: signer,
```

Comment:
nit: Use &signer instead of signer

Reply:
`vm: signer` is what we are doing today (left view, L102), should i change it?